### ISSUE TITLE
Support active aur helper

### ISSUE THREAD

**Issue by frozenxuan** [aarnt/octopi#331]
All 3 currently supported aur helpers (pacaur, yaourt, trizen) are either discontinued or problematic (source: https://wiki.archlinux.org/index.php/AUR_helpers). Please support active aur helper such as aurman instead.

**Comment by michaldybczak** [comment]
Trizen is currently in active development and works rather well. Its support in Octopi was added quite recently. Some claim they saw some yaourt commits so the rumors of its demise may be premature. Pacaur still works great (the best out of the 3 options IMO). It may not be so forever but currently this is still a viable choice.

I'm afraid that AUR helpers support requires time and lot of work to implement and that is why only popular ones are added, but I do would like to see more of them being integrated with Octopi. Maybe will see it once day. Aurman would be surely an interesting one. I switched to trizen but I hate its password timeout which I always miss and that results with failed build. I submitted that as an issue for trizen but will see if it will be somehow resolved.
